fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/serialization/LoanEventApiJsonValidator.java

```diff
@@ -224,7 +228,7 @@ public void validateNewRepaymentTransaction(final String json) {
 
         final Set<String> transactionParameters = new HashSet<>(
                 Arrays.asList("transactionDate", "transactionAmount", "externalId", "note", "locale", "dateFormat", "paymentTypeId",
-                        "accountNumber", "checkNumber", "routingCode", "receiptNumber", "bankNumber", "loanId"));
+                        "accountNumber", "checkNumber", "routingCode", "receiptNumber", "bankNumber", "loanId", "fullChargeback"));
```

Comment:
@josehernandezfintecheandomx I dont think we need the "fullChargeback" field support here... the UI cannot tell whether it is full chargeback or not...

Reply:
Done, removed